recipes/boxsdk/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "boxsdk" %}
+{% set version = "3.9.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/boxsdk-{{ version }}.tar.gz
+  sha256: 10e23e2f82e9cff2b2e501b7ca7ffe7bac0e280d1cd4b2983dea95f826e3008b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pytest
```

Comment:
why do you need ``pytest` in host?

Reply:
Thank you for the review.  I was just taking what `grayskull` generated.  Removed.